Xamarin.Forms.Core/Accessibility.cs

```diff
@@ -0,0 +1,14 @@
+﻿
+namespace Xamarin.Forms
+{
+	public class Accessibility
+	{
+		public static readonly BindableProperty HintProperty = BindableProperty.Create("Hint", typeof(string), typeof(Element), default(string));
```

Comment:
does it makes sense to only allow that on `VisualElement`? why do we need `Element` for ? `Cells` ?